topics/variant-analysis/tutorials/aiv-analysis/tutorial.md

```diff
@@ -75,7 +75,10 @@ Instead, to save some time, you can copy an existing shared history on Galaxy Eu
 
 > <hands-on-title>Prepare the Galaxy history</hands-on-title>
 >
-> 1. Open the [shared history](https://usegalaxy.eu/u/wolfgang-maier/h/influenza-resources) with pre-prepared INSAFlu reference data
+> 1. On Galaxy Europe, open the [shared history](https://usegalaxy.eu/published/history?id=0fc6b637aadbc8a2) with pre-prepared INSAFlu reference data.
```

Comment:
There is a way to annotate this as metadata have you tried that?

Reply:
For zenodo data I know that, but for shared histories?